plugin/loinc-transformation-maven-plugin/src/main/java/dev/ikm/maven/LoincTransformationMojo.java

```diff
@@ -603,14 +603,17 @@ private void createAxiomSemanticsLoincConcept(Session session, EntityProxy.Conce
     private void createLoincClassSemantic(Session session, EntityProxy.Concept concept,
                                           String loincClass, String loincClassType) {
         EntityProxy.Pattern loinClassPattern = LoincUtility.getLoincClassPattern(namespace);
+        int classTypeInt = Integer.parseInt(loincClassType);
+        String loinClassPartNumber = LoincUtility.getPartNumberFromCache(loincClass.toLowerCase(), "CLASS");
+        EntityProxy.Concept loincClassPartConcept = LoincUtility.makeConceptProxy(namespace, loinClassPartNumber);
         try {
             session.compose((SemanticAssembler assembler) -> {
                 assembler.semantic(EntityProxy.Semantic.make(PublicIds.of(UuidT5Generator.get(namespace, concept.publicId().asUuidArray()[0] + loincClass))))
                         .reference(concept)
                         .pattern(loinClassPattern)
                         .fieldValues(fv -> fv
-                                .with(loincClass)
-                                .with(loincClassType)
+                                .with(loincClassPartConcept)
+                                .with(classTypeInt)
```

Comment:
changing from a String to int - I'm assuming this is the intention, and we are able to successfully parse the csv's and all members of that column are integers.